todo.js

```diff
@@ -15,3 +15,39 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+const fs = require('fs');
+let todoList = fs.readFileSync("todos.txt","utf-8");
```

Comment:
It will be easier if you represent the todo list as an array of strings, each one representing a task, rather than a single giant string that you have to spelunk through to get work done.